code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -191,17 +202,22 @@
 		"Boxes" = list(
 		/obj/item/ammo_magazine/box9mm = 100,
 		/obj/item/ammo_magazine/acp = 100,
+		/obj/item/ammo_magazine/acp/polymer = 100,
 		/obj/item/ammo_magazine/magnum = 100,
+		/obj/item/ammo_magazine/magnum/polymer = 100,
 		/obj/item/ammo_magazine/box10x24mm = 100,
+		/obj/item/ammo_magazine/box10x24mm/polymer = 100,
 		/obj/item/ammo_magazine/box10x26mm = 100,
+		/obj/item/ammo_magazine/box10x26mm/polymer = 100,
 		/obj/item/ammo_magazine/box10x27mm = 100,
+		/obj/item/ammo_magazine/box10x27mm/polymer = 100,
 		/obj/item/storage/box/visual/magazine = 30,
 		),
-	)
 
 	seasonal_items = list(
-		SEASONAL_GUNS = "Seasonal",
+		SEASONAL_GUNS = "seasonal_guns",
 	)
+)
 
 /obj/machinery/vending/marine/shared/hvh
```

Comment:
Currently this is putting the `seasonal_items` list inside the `products` list above, since the latter only ends at line 220.
Reverting this change should fix it, or manually setting it to:
```dm
        /obj/item/storage/box/visual/magazine = 30,
        ),
    )

    seasonal_items = list(
        SEASONAL_GUNS = "Seasonal",
    )

/obj/machinery/vending/marine/shared/hvh
```